pylabnet/hardware/staticline/staticline.py

```diff
@@ -150,6 +150,12 @@ def _setup_NiDaqMxDriver(self, **kwargs):
         # Log successfull setup.
         self.log.info(f"NiDaq {self.hardware_module.dev} output {ao_output} successfully assigned to staticline {self.name}.")
 
+    def _setup_toptica(self, **kwargs):
+
+        self.up = lambda: self.hardware_module.turn_on()
+        self.down = lambda: self.hardware_module.turn_off()
```

Comment:
@cknaut will this work? should we try writing a launcher for all relevant static-lines for the current experiment (what else is there, just the imaging shutter?), and testing?

Reply:
Almost, just need to register the driver module name in the following line in the `init` function.

```python

        # Dictionary listing all hardware modules which can address
        # staticlines and their corresponding setup functions.
        registered_staticline_modules = {
            'zi_hdawg':  self._setup_HDWAGDriver,
            'nidaqmx_card': self._setup_NiDaqMxDriver
        }
```

Problem with this staticline class is that it cannot check if the staticline-toggle was successful, which in turn can yield a scenario where the position of the button is not indicative of the actual state of the staticline. Added an issue (#82).